app/static/responsive.css

```diff
@@ -35,6 +35,27 @@
     display: none;
   }
 
+  .show-menu {
+    display: block;
+    position: absolute;
+    top: 85px;
+    line-height: 58px;
+    width: 335px;
+    z-index: auto;
+    height: 100%;
+    background-color: #6070ff;
+    margin-top: -85px;
+    text-align: start;
+    opacity: 0.8;
+  }
```

Comment:
- [ ] Your menu looks quite similar to the Figma template, but the space from the first menu option to the top is not the same as the Figma template. Please add the correct space above the options.